eng/performance/scenarios.proj

```diff
@@ -100,6 +100,34 @@
     </HelixWorkItem>
   </ItemGroup>
 
+  <!-- SOD & Device Startup for Windows Desktop MAUI -->
+  <ItemGroup>
+    <HelixWorkItem Include="SOD - Desktop Maui Unpackaged" Condition="'$(TargetsWindows)' == 'true'">
+      <PayloadDirectory>$(ScenariosDir)mauidesktop</PayloadDirectory>
+      <PreCommands>$(Python) pre.py publish -c Release -f $(PERFLAB_Framework)-windows10.0.19041.0</PreCommands>
+      <Command>$(Python) test.py sod --scenario-name &quot;%(Identity)&quot;</Command>
+    </HelixWorkItem>
+    <HelixWorkItem Include="Device Startup - Desktop Maui Default" Condition="'$(TargetsWindows)' == 'true'">
```

Comment:
If you want to avoid the duplicate `Condition`, you can move it up to the `<ItemGroup>`:

```suggestion
  <ItemGroup Condition="'$(TargetsWindows)' == 'true'">
    <HelixWorkItem Include="SOD - Desktop Maui Unpackaged">
      <PayloadDirectory>$(ScenariosDir)mauidesktop</PayloadDirectory>
      <PreCommands>$(Python) pre.py publish -c Release -f $(PERFLAB_Framework)-windows10.0.19041.0</PreCommands>
      <Command>$(Python) test.py sod --scenario-name &quot;%(Identity)&quot;</Command>
    </HelixWorkItem>
    <HelixWorkItem Include="Device Startup - Desktop Maui Default">
```